pkgs/applications/networking/calls/default.nix

```diff
@@ -68,14 +74,15 @@ stdenv.mkDerivation rec {
     "-Dgtk_doc=false"
   ];
 
-  doCheck = true;
+  doCheck = false;
```

Comment:
There was a new check added in the last update called manager and I get this error, that I don't know how to fix:
```
--- command ---
19:26:54 GSETTINGS_BACKEND='memory' G_TEST_SRCDIR='/build/source/tests' MALLOC_CHECK_='2' G_TEST_BUILDDIR='/build/source/build/tests' G_DEBUG='gc-friendly,fatal-warnings' PYTHONDONTWRITEBYTECODE='yes' /build/source/build/tests/manager
--- stdout ---
# random seed: R02S083552078a96d849a5575aea64448b9d
1..3
# Start of Calls tests
# Start of Manager tests
Bail out! GLib-GIO-FATAL-ERROR: No GSettings schemas are installed on the system
--- stderr ---

(/build/source/build/tests/manager:2496): GLib-GIO-ERROR **: 19:26:54.365: No GSettings schemas are installed on the system
-------
```

Reply:
@craigem since you wrote the checkPhase, do you know how to pass gsettings schema through xvfb-run or generally fix this issue? I can't find any method of doing so and I would like to get the tests to run.

Reply:
@Pacman99 as far as I understand gsettings schemas are searched for in `XDG_DATA_DIRS`, maybe setting that will help.

Reply:
I tried that, but it doesn't get forwarded by xvfb-run. Or maybe I'm doing something wrong.

Reply:
Yeah I was just reading the errors wrong, setting `XDG_DATA_DIRS` to folks gsettings-schema fixed the issue.